lotto/offchain/src/Lotto.hs

```diff
@@ -173,7 +173,7 @@ mintSeal outRef value = do
 -- contains the seal and the value gambled, nothing more.
 -- Typically one has to propagate the value of the previous lotto input his or herself.
 splay ::
-  Cooked.MonadBlockChainWithoutValidation m =>
+  Cooked.MonadBlockChain m =>
```

Comment:
Why do we loosen this constraint? Is `slotRangeBefore` not available in `MonadBlockChainWithoutValidation`?